package.json

```diff
@@ -33,7 +36,6 @@
     "lint:astro": "astro check",
     "lint:eslint": "eslint . --ext .js,.ts,.astro",
     "lint:html": "html-validate --config config/htmlvalidate/config.json dist/",
-    "seed:docs": "jiti config/datocms/scripts/seed-docs.ts",
```

Comment:
I like having this one available for Head Start users, same as other seed scripts. But I guess having it in the workspace only might work just as well, as it still needs it's own `npm install`.